radio/src/telemetry/flysky_nv14.cpp

```diff
@@ -125,30 +125,41 @@ int32_t GetSensorValueFlySkyNv14(const FlyskyNv14Sensor* sensor,
   int32_t value = 0;
   const nv14SensorData* sensorData =
       reinterpret_cast<const nv14SensorData*>(data + sensor->offset);
+  static bool oldRmFm = false;
+
   if (sensor->bytes == 1)
     value = sensorData->UINT8;
   else if (sensor->bytes == 2)
     value = sensor->issigned ? sensorData->INT16 : sensorData->UINT16;
   else if (sensor->bytes == 4)
     value = sensorData->UINT32;
 
+  if (NV14internalModuleFwVersion <= 0x1000E) {
+    if (sensor->id == FLYSKY_SENSOR_RX_SIGNAL) {
```

Comment:
Is the two stage check is needed because there are two firmwares that report the same version number?

Reply:
Yes, exactly: there are two FWs with the same version and different behavior.